tests/test_sql_tools.py

```diff
@@ -76,97 +74,52 @@ def test_create_locality(self):
         """testing create_locality function by
            recreating insert protocol for locality table, but with sqlite DB"""
 
```

Comment:
Shortened unittest for locality insert statement, using new sql_csv_utils class.